Makefile

```diff
@@ -46,7 +46,14 @@ OBJS += $(ASM_SOURCES:%.S=%.o)
 
 TARGET := kernel64.bin
 
+# On Linux systems, we build directly. On non-Linux, we rely on the 'docker%'
+# rule below to create an Ubuntu container and perform the Linux-specific build
+# steps therein.
+ifeq ($(SYSTEM), LINUX)
 all: $(TARGET)
+else
+all: docker$(TARGET)
```

Comment:
does this need a colon: `docker:$(TARGET)`?

Reply:
see above